cypress/support/pages/admin_console/components/TabPage.ts

```diff
@@ -0,0 +1,38 @@
+import CommonElements from "../../CommonElements";
+
+export default class TabPage extends CommonElements {
+  tabItemSelector;
```

Comment:
Can we add an access modifier and type here?
```suggestion
  protected tabItemSelector: string;
```